tests/e2e/tests/pilot/istio_rbac_test.go

```diff
@@ -162,3 +175,57 @@ func TestRBAC(t *testing.T) {
 		}
 	}
 }
+
+func TestRBACForEgressGateway(t *testing.T) {
+	cfgs := setupRbacRules(t, []string{
```

Comment:
A quick comment what/why all these policies are for would be helpful for the reader.

Reply:
Updated to a single file with better comments.